monitoring/bin/deploy_monitoring_tenant.sh

```diff
@@ -195,7 +195,7 @@ deploy_tenant_dashboards monitoring/multitenant/dashboards
 
 # If a deployment with the old name exists, remove it first
 if helm3ReleaseExists "v4m-tenant-$VIYA_TENANT" $MON_NS; then
-  log_verbose "Removing outdated instance of SAS Viya Monitoring"
+  log_verbose "Removing outdated [${VIYA_NS}/${VIYA_TENANT}] SAS Viya Monitoring Helm chart release"
```

Comment:
I believe the message with come out looking like:
_**Removing outdated [d12345/acme] SAS Viya Monitoring Helm chart release**_
I think reworking it a bit might make it more readable, perhaps something like:
**_Removing outdated SAS Viya Monitoring Helm chart release for tenant [d12345/acme]_**